src/main/java/mods/railcraft/common/items/enchantment/EnchantmentBoost.java

```diff
@@ -0,0 +1,37 @@
+/*------------------------------------------------------------------------------
+ Copyright (c) CovertJaguar, 2011-2017
+ http://railcraft.info
+
+ This code is the property of CovertJaguar
+ and may only be used with explicit written
+ permission unless otherwise specified on the
+ license page at http://railcraft.info/wiki/info:license.
+ -----------------------------------------------------------------------------*/
+package mods.railcraft.common.items.enchantment;
+
+import net.minecraft.inventory.EntityEquipmentSlot;
+
+/**
+ * Boost enchantment.
+ */
+public class EnchantmentBoost extends EnchantmentToolRailcraft {
+
+    public EnchantmentBoost(Rarity rarity) {
+        super("boost", rarity, EntityEquipmentSlot.MAINHAND);
+    }
+
+    @Override
+    public int getMinEnchantability(int level) {
+        return 9 + level * 8;
```

Comment:
http://minecraft.gamepedia.com/Enchanting/Levels
The levels are composite, as the page referenced says in the red box, "These are not the levels you need to pay to be able to receive specific enchantments, you must also take into account the modification of levels that occur in step one."